servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -1129,6 +1129,13 @@
         ],
         "parameters": [
           {
+            "name": "timeout",
+            "in": "query",
+            "description": "The timeout in seconds",
+            "type": "integer",
+            "required": false
```

Comment:
The default is determined by the server side in the config. Client would not be able to know.